platform/api/brightbox/api.go

```diff
@@ -220,7 +220,7 @@ func (a *API) UploadImage(ctx context.Context, name, URL string) (string, error)
 	}
 
 	// It usually takes around 20 seconds to extract the image.
-	if err := util.WaitUntilReady(2*time.Minute, 5*time.Second, func() (bool, error) {
+	if err := util.WaitUntilReady(4*time.Minute, 5*time.Second, func() (bool, error) {
```

Comment:
```suggestion
	if err := util.WaitUntilReady(5*time.Minute, 5*time.Second, func() (bool, error) {
```
How about 5 or even 10 to make sure we don't hit this occasionally?